Tests/TensorFlowTests/InitializerTests.swift

```diff
@@ -170,6 +170,17 @@ final class InitializerTests: XCTestCase {
         testDistribution(t, expectedMean: 0, expectedStandardDeviation: stdDev)
     }
 
+    func testCategoricalFromLogits() {
+        let probabilities = Tensor<Float>([[0.5, 0.3, 0.2], [0.6, 0.3, 0.1]])
+        let logits = log(probabilities)
+        let t = Tensor<Int32>(randomCategorialLogits: logits, sampleCount: 1)
+
+        XCTAssertEqual(TensorShape([2, 1]), t.shape)
+        // Test all elements are in range of [0, 3)
+        XCTAssertEqual(1, (Tensor<Int32>(t .>= Tensor<Int32>([[0], [0]])).product()).scalar!)
+        XCTAssertEqual(1, (Tensor<Int32>(t .< Tensor<Int32>([[3], [3]])).product()).scalar!)
```

Comment:
It would be a bit simpler to use `.all()` instead of `.product()`, like this: `XCTAssertTrue((t .>= Tensor<Int32>([[0], [0]])).all())`